chore: internal docs for beta (#187)
storyblok/storyblok-cli: README.md

## Documentation

> [!WARNING]

> > Official documentation about this package v4 is not yet available. Please refer to the internal documentation for the [v4 beta](src/README.md).

Reviewer: ```suggestion
> Official documentation for this package v4 is still in development. In the meantime, please refer to the internal documentation for the [v4 beta](src/README.md).
```

> ~~For complete documentation, please visit [https://storyblok.com/docs](https://storyblok.com/docs/packages/my-joyful-package)~~

Reviewer: I would comment this out until we have the actual documentation link.
